home/models.py

```diff
@@ -377,6 +373,17 @@ class User(AbstractUser):
         }
     )
 
+    class Meta:
+        # planetterp admins are a level between staff users and normal users:
+        # admins can view the admin panel and take all actions theiren, but
+        # cannot view the django admin panel.
+        # Essentially, this role is for site admins which should not have access
+        # to the prod db, which the django admin panel grants to a moderate
+        # degree.
+        permissions = [
+            ("admin", "Can take any planetterp admin actions")
```

Comment:
"moderator" is more fitting here

Reply:
I've gone with `mod`, as `moderator` was quite long.